crates/serpen/tests/fixtures/comprehensive_ast_rewrite/core/utils/helpers.py

```
@@ -0,0 +1,111 @@
+"""
+Utility helpers module with extensive naming conflicts
+"""
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)